storybooknative/src/CenterView/index.js

```diff
@@ -4,13 +4,16 @@ import * as React from "react";
 import { View } from "react-native";
 
 import styled from "../styled";
+import type { StylePropType } from "../types/StylesPropType";
 
-const Main = styled(View, {
+const style: StylePropType = {
   flexGrow: 1,
   justifyContent: "center",
   alignItems: "center",
   backgroundColor: "#f5fcff",
-});
+};
+
+const Main = styled(View, style);
```

Comment:
also it could be used like: 
```
const Main = styled(
  View,
  ({
    flexGrow: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#f5fcff",
  }: StylePropType),
);
```

both is fine by me

Reply:
I find this
```
const Main = styled(
  View,
  ({
    flexGrow: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#f5fcff",
  }: StylePropType),
);
``` 
less readable, so I will stick with first option.